Doc/using/cmdline.rst

```diff
@@ -1076,9 +1076,9 @@ conflict.
 .. envvar:: PYTHONCPUCOUNT
 
    If this variable is set to a positive integer, it overrides
-   :func:`os.cpu_count` and and :func:`os.process_cpu_count`.
+   :func:`os.cpu_count` and and :func:`os.process_cpu_count` result.
```

Comment:
In init_config, you used "result". I would prefer consistency, either "result" in both docs, or "return value" in both docs.

```suggestion
   :func:`os.cpu_count` and and :func:`os.process_cpu_count` return value.
```